dex/wait/queue.go

```diff
@@ -253,14 +257,15 @@ func (q *TaperingTickerQueue) Run(ctx context.Context) {
 			sort.Slice(waiters, func(i, j int) bool {
 				return waiters[i].nextTick.Before(waiters[j].nextTick) // ascending, next tick first
 			})
-			// NOTE: timer leaked until it fires - consider NewTimer and Stop here
```

Comment:
Attempting to address this while we are here (it's in a separate commit - could easily revert it or change).

Reply:
I think `maybeStop` will be called on next `for` iteration (since there will be at least 1 elem in `waiters`), so no need to call it here as well